hack/app_sre_create_image_catalog.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+set -exv
+
+# prefix var with _ so we don't clober the var used during the Make build
+# it probably doesn't matter but we can change it later.
+_OPERATOR_NAME="route-monitor-operator"
+
+BRANCH_CHANNEL="$1"
+QUAY_IMAGE="$2"
+
+GIT_HASH=$(git rev-parse --short=7 HEAD)
+GIT_COMMIT_COUNT=$(git rev-list $(git rev-list --max-parents=0 HEAD)..HEAD --count)
+
+# clone bundle repo
+SAAS_OPERATOR_DIR="saas-route-monitor-operator-bundle"
+BUNDLE_DIR="$SAAS_OPERATOR_DIR/route-monitor-operator/"
+
+rm -rf "$SAAS_OPERATOR_DIR"
+
+git clone \
+    --branch "$BRANCH_CHANNEL" \
+    https://app:"${APP_SRE_BOT_PUSH_TOKEN}"@gitlab.cee.redhat.com/service/saas-route-monitor-operator-bundle.git \
+    "$SAAS_OPERATOR_DIR"
+
+# remove any versions more recent than deployed hash
+REMOVED_VERSIONS=""
+if [[ "$REMOVE_UNDEPLOYED" == true ]]; then
+    DEPLOYED_HASH=$(
+        curl -s "https://gitlab.cee.redhat.com/service/app-interface/raw/master/data/services/osd-operators/cicd/saas/saas-${_OPERATOR_NAME}.yaml" | \
+            docker run --rm -i quay.io/app-sre/yq yq r - "resourceTemplates[*].targets(namespace.\$ref==/services/osd-operators/namespaces/hivep01ue1/${_OPERATOR_NAME}.yml).ref"
+    )
+
+    delete=false
+    # Sort based on commit number
+    for version in $(ls $BUNDLE_DIR | sort -t . -k 3 -g); do
+        # skip if not directory
+        [ -d "$BUNDLE_DIR/$version" ] || continue
+
+        if [[ "$delete" == false ]]; then
+            short_hash=$(echo "$version" | cut -d- -f2)
+
+            if [[ "$DEPLOYED_HASH" == "${short_hash}"* ]]; then
+                delete=true
+            fi
+        else
+            rm -rf "${BUNDLE_DIR:?BUNDLE_DIR var not set}/$version"
+            REMOVED_VERSIONS="$version $REMOVED_VERSIONS"
+        fi
+    done
+fi
+
+# generate bundle
+PREV_VERSION=$(ls "$BUNDLE_DIR" | sort -t . -k 3 -g | tail -n 1)
+
+./hack/generate-operator-bundle.py \
```

Comment:
@georgettica and I worked yesterday in an approach that didn't need this script anymore: using `make bundle` and copying the `bundle/manifests` file

Reply:
not exactly, as you mentioned we need the `replaces` filed to be calculated as we want. so we can just override in python for now

Reply:
I understood you were working in the manifest generation using `make bundle`. Hence my comment